code/model/modelsinc.h

```diff
@@ -12,6 +12,9 @@
 
 
 class polymodel;
+struct vec3d;
+struct matrix;
+typedef std::uint8_t ubyte;
```

Comment:
We already have this typedef in `pstypes.h` to please only use that instead of repeating it here.

Reply:
This is also part of the problem with this specific header, whiout that you are going to get errors about missing these types in CI. For the record these types were already in use by the header, as i only added a definition.

Reply:
Using pragma message this header now works, but it will fail again if i remove those forward declarations.